src/web/components/Card/components/CardFooter.tsx

```diff
@@ -47,7 +48,12 @@ export const CardFooter = ({
 	mediaMeta,
 	commentCount,
 	isFullCardImage,
+	labBadge,
 }: Props) => {
+	if (format.theme === Special.Labs && labBadge) {
+		return labBadge;
```

Comment:
```suggestion
		return (
			<footer>labBadge</footer>
		);
```

for the semantic html win